src/components/HorizontalTabList/index.js

```diff
@@ -1,40 +1,24 @@
 import React, {Component} from 'react';
 import './styles.css';
 
-class HorizontalTabList extends Component {
-
-  getListItems() {
-    const list = this.props.list;
-    const listItems = [];
-
-    for (let i = 0; i < list.length; i++) {
-      const newItem = this.generateListItem(list[i].symbol, i);
-      listItems.push(newItem);
-    }
-
-    return listItems;
-  }
-
-  generateListItem(title, counter) {
-    if (title === this.props.selectedItem.symbol) {
-      return <li className="list-inline-item active" key={counter}>{title}</li>;
-    } else {
-      return <li className="list-inline-item" key={counter} onClick={() => this.changeSelectedItem(title)}>{title}</li>;
-    }
-  }
-
-  changeSelectedItem(title) {
-    this.props.changeSelected(title);
-  }
-
-  render() {
-    return (
-      <ul className="list-inline tablist">
-        {this.getListItems()}
-      </ul>
-    )
-  }
-
-}
-
-export default HorizontalTabList;
+export default class HorizontalTabList extends Component {
+  getListItems = () =>
+    this.props.list.map((item, i) => this.generateListItem(item.symbol, i));
+
+  generateListItem = (title, counter) => (
```

Comment:
This should be a separate component as well (please see my comment on the `generateButton` method below, it's also about performance).